packages/noco-docs/docs/100.data-sources/020.connect-to-data-source.md

```diff
@@ -23,6 +23,14 @@ To connect to an external data source, follow the steps below:
    | Database      | Name of the database to connect to                                                   |
    | Schema name   | Name of the schema to connect to                                                     |
 
+4a. Optionally, if the connection required is TLS/MTLS for MITM protection, follow these additional steps below:
+
+   - Click on `SSL & Advanced Parameters`, \
+   - Select `SSL Mode` and upload the client certificate, client key, and Root CA files by clicking on the file.    
+   \
+   Note: The entire connection parameters can be viewed in json form by clicking on the "Edit connection JSON" and edited as required by database server type.\
+   Example: In PostgreSQL with SSL mode set to "Required-Identity," if the server certificate's common name (cname) differs from the actual DNS/IP used for connection, the connection may fail. To resolve this, add a "servername" property with the cname value under the SSL section, along with certificate information. Additional details are available in the Knex configuration options.[knex configuration options](https://knexjs.org/guide/#configuration-options)
```

Comment:
1. better have a line break between note and example. they look too packed.
2. `options.[knex configuration options](https://knexjs.org/guide/#configuration-options)` . Maybe just `[Knex configuration options](https://knexjs.org/guide/#configuration-options)` for better UI.

Reply:
1. "\" does add line break while rendering. For readability let me add it in source as well. 
2. ack. good catch.

Reply:
1. maybe add one more. you can see the above screenshot. two paragraphs are packed to each other.

Reply:
although not exactly what you suggested here. I slightly reworded to reduce verbosity. Below is how it looks now

source view : 
![image](https://github.com/nocodb/nocodb/assets/113826417/24328be3-e5c0-4d6a-af5d-e57bbdeb538c)

rendered view: 
![image](https://github.com/nocodb/nocodb/assets/113826417/7325eba8-ae94-4fa7-ad9b-a6a25da1a97c)

Reply:
Looks good now.